package.json

```diff
@@ -48,6 +48,11 @@
     "exclude": [
       "dist/**/*.spec.js",
       "dist/**/*.d.ts"
+    ],
+    "all": true,
+    "reporter": [
```

Comment:
Is this needed since the reporters are specified in the gulp task? If these can replace that then I am okay with this as long as they are removed from the gulp task.